pkgs/misc/uboot/default.nix

```
@@ -48,7 +47,8 @@ let
     src = if src == null then defaultSrc else src;
 
     patches = [
-      ./0001-configs-rpi-allow-for-bigger-kernels.patch
+      ./0001-rpi.env-allow-for-bigger-kernels.patch
+      ./0002-rpi.env-try-to-boot-from-mmc0-first.patch
```

To keep track of this new situation.

```suggestion
      # https://patchwork.ozlabs.org/project/uboot/patch/20231007155535.1699567-1-benwolsieffer@gmail.com/
      ./0002-rpi.env-try-to-boot-from-mmc0-first.patch
```